app/commands/payments/paypal/subscription/create.rb

```diff
@@ -3,9 +3,9 @@
 class Payments::Paypal::Subscription::Create
   include Mandate
 
-  initialize_with :user, :id, :amount, :product, :interval
+  initialize_with :user, :id, :amount, :product, :interval, status: :active
```

Comment:
The added `status` field is there to be able to initialize a subscription with the `pending` status, which we need when manually initiating a paypal subscription.